src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
It is not obvious to me that these are global shared objects, if you import them with the `import` statement. It feels to me like we want to hide something.. We should be honestly and obviously deal with a global-object, so the developer is aware of that.

It is/was everything prepared on the UI side, serial (ordered) loading and obvious shared objects between the plugins. 
This is now gone and it is a regression to me, because it did not any harm.
Actually the approach followed the keep-it-simple-strategy, imho.
Yes, the next step would be to deliver this in an ordered way from the service layer, but this isn't a must to develop our first plugins and would be for sure one of the next steps.